src/utils/helper-functions.tsx

```diff
@@ -288,3 +288,13 @@ export const getGroupedByDate = <T extends { createdAt: string }>(
 
     return result
   }, [])
+
+export const getAttachmentInfo = (
```

Comment:
I suggest to rename this function to someting like `parseFilename` to help our teammates in finding this method and reuse in other places

Reply:
solved